### ISSUE TITLE
Storage bundled when unused

### ISSUE THREAD

**Issue by dvlden** [wxt-dev/wxt#238]
### Describe the bug

New feature from `wxt/browser` - `storage` is bundled in `background` script/worker, even when it's not used at all.

### To Reproduce

Bundle a project that doesn't use storage in background script and check `.output/*/background.js`

### Expected behavior

Should not contain any code related to storage (e.g search for `unstorage`).

### Environment
```
System:
    OS: macOS 14.1
    CPU: (12) arm64 Apple M2 Max
    Memory: 67.13 GB / 96.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.3.1 - ~/.nvm/versions/node/v20.3.1/bin/node
    Yarn: 1.22.21 - ~/.nvm/versions/node/v20.3.1/bin/yarn
    npm: 10.1.0 - ~/.nvm/versions/node/v20.3.1/bin/npm
    pnpm: 8.6.12 - ~/.nvm/versions/node/v20.3.1/bin/pnpm
    bun: 1.0.10 - /opt/homebrew/bin/bun
  Browsers:
    Chrome: 119.0.6045.123
    Safari: 17.1
  npmPackages:
    wxt: ^0.10.0 => 0.10.0
```

**Comment by aklinker1** [comment]
Hmm, it should be tree-shakable... Ahh, I know what the problem is. I should use a dynamic import when adding it to the global scope.

I guess Vite thinks importing `wxt/storage` has a side effect, which makes sense since it is running code to construct the `storage` variable

**Comment by aklinker1** [comment]
Fixed by https://github.com/wxt-dev/wxt/commit/3da3e07bda927fef022e32fbb5cca9d9d3c92415. Couldn't figure out why tree-shaking wasn't working, so I removed the runtime `wxt` helper.

**Comment by aklinker1** [comment]
Released in [`v0.10.1`](https://github.com/wxt-dev/wxt/releases/tag/v0.10.1)
